src/specs/data_structures.md

```diff
@@ -995,3 +995,15 @@ Various [consensus parameters](consensus.md#system-parameters) are committed to
 | `availableDataOriginalSquareMax` | `uint64`                              | The `AVAILABLE_DATA_ORIGINAL_SQUARE_MAX`. |
 
 In order to compute the `consensusHash` field in the [block header](#header), the above list of parameters is [hashed](#hashing).
+
```

Comment:
Done.

Reply:
If we put StateFraudProof to networking.md, shouldn't we also put BadEncodingFraudProof there as well?

Reply:
ref: https://github.com/celestiaorg/celestia-specs/pull/191

Reply:
Sorry, I meant _just_ put it in `networking.md`, not _also_. So #191 is not needed as a separate PR, just change this one.

Both that state transition fraud proof and bad encoding fraud proofs should be in `networking.md` since they're only distributed at the p2p layer and never included in blocks.

Reply:
I moved both fraud proofs to networking.md.